### PR TITLE
feat(shopify): add multilanguage support

### PR DESCRIPTION
**Pull request by cbernat**
- Added multilanguage support
- Fixed tests in consequence

image

On general error added button to go back
image

### CODE REVIEW

File: src/components/Integrations/Shopify/Shopify.js [hunk 1]
```diff
@@ -4,20 +4,22 @@ import Loading from '../../Loading/Loading';
...
   const [error, setError] = useState(null);
   const [isLoading, setIsLoading] = useState(true);
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
 
   useEffect(() => {
     const getData = async () => {
       const result = await shopifyClient.getShopifyData();
       if (result.expectedError && result.expectedError.cannotConnectToAPI) {
-        setError('Error: No hemos podido conectar con la Api de Shopify, vuelve a intentar luego.');
+        setError(_('shopify.error_cannot_access_api'));
```
Comment (andresmoschini) [inline src/components/Integrations/Shopify/Shopify.js]: Creo que para los errores no es necesario traducir, probaste lo siguiente?

```suggestion
        setError('shopify.error_cannot_access_api');
```

Comment (cbernat) [reply src/components/Integrations/Shopify/Shopify.js]: I will give this a try, I'm not so sure.

Comment (cbernat) [reply src/components/Integrations/Shopify/Shopify.js]: The translation is needed here. I cannot remove it like you suggested.

Comment (andresmoschini) [reply src/components/Integrations/Shopify/Shopify.js]: Oh, I was completely sure that it was not, sorry.

Comment (andresmoschini) [reply src/components/Integrations/Shopify/Shopify.js]: Are you sure? 

https://github.com/FromDoppler/doppler-webapp/blob/28bb56bb19802e1e2734a750ca0e960b5292a00a/src/components/form-helpers/form-helpers.js#L167

Comment (cbernat) [reply src/components/Integrations/Shopify/Shopify.js]: I do not use the form helper here. I have no forms in this integration. SetError is just a hook.

**Comment by cbernat** [comment]
Test this please.

File: src/components/Integrations/Shopify/Shopify.js (hunk 1 above)
Comment (andresmoschini) [reply src/components/Integrations/Shopify/Shopify.js]: Ohh, sorry, my bad.

**Comment by cbernat** [comment]
Test this please.

**Pull request merged**